### DISCUSSION TITLE
TuyaWebPlatform might not be ready for Homebridge v2.0

### DISCUSSION THREAD

**Discussion by clossie** [milo526/homebridge-tuya-web#617] [General]
The developer has not specifically marked this plugin as compatible with Homebridge v2.0, but it may still work.

Has anyone else started seeing this warning ? Are there works ongoing to certify Tuya web for Homebridge V2.0?

**Comment by milo526** [comment]
Hello,
I am interested in getting this plugin compatible with Homebridge v2.0, but I'm very limited in my available time.
If someone could tell me if they experience any problems I'd be happy to take a look at them.

**Comment by yayfrogs** [comment]
I have all of three Tuya-backed smart plugs, so this is a very limited test, but they're working on v2.0.0-beta.29.

**Comment by clossie** [comment]
Homebridge v2.0.0-beta.29
Ui 5.2.0
Node 22.17.1

Items working without issue via the SMART Life App.
Nedis Bulbs - Including Dimming capabilities and color change
Link2Home Bulbs - Non Dimmable / Non Color Changing
Teckin Bulbs - Non Dimmable / Non Color Changing
Teckin Plugs (UK Spec)
Amazon Basic Bulbs - - Non Dimmable / Non Color Changing
GY Smart Bulb -  Including Dimming capabilities and color change

**Comment by glitch82** [comment]
> Homebridge v2.0.0-beta.29
> 
> Ui 5.2.0
> 
> Node 22.17.1
> 
> 
> 
> Items working without issue via the SMART Life App.
> 
> Nedis Bulbs - Including Dimming capabilities and color change
> 
> Link2Home Bulbs - Non Dimmable / Non Color Changing
> 
> Teckin Bulbs - Non Dimmable / Non Color Changing

That's unfortunate, these used to work just fine with earlier versions of either Homebridge, the plugin, or both. 

> Teckin Plugs (UK Spec)
> 
> Amazon Basic Bulbs - - Non Dimmable / Non Color Changing
> 
> GY Smart Bulb -  Including Dimming capabilities and color change
> 
>
